packages/twenty-front/src/modules/ui/utilities/screen-size/hooks/useScreenSize.ts

```diff
@@ -0,0 +1,25 @@
+import { useEffect, useState } from 'react';
+
+export const useScreenSize = () => {
```

Comment:
@abdul-irfan-k introducing this hook as based on design, it does not seem to be possible to handle this by css only